mods/digimon/abilities.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+exports.BattleAbilities = {
+	"data": {
+		id: "data",
+		name: "Data",
+		onBasePowerPriority: 8,
+		onBasePower: function (basePower, attacker, move) {
+			if (attacker.ability) {
+				if (attacker.ability === 'vaccine') {
+					this.debug('Data boost');
+					return this.chainModify(1.5);
+				}
+			}
+		},
+		onBasePowerPriority: 8,
+		onBasePower: function (basePower, attacker, move) {
```

Comment:
Duplicate keys `onBasePowerPriority` and `onBasePower`, combine the functions into one.